runtime/doc/fold.txt

```diff
@@ -595,7 +595,8 @@ A closed fold is indicated with a '+'.
 These characters can be changed with the 'fillchars' option.
 
 Where the fold column is too narrow to display all nested folds, digits are
-shown to indicate the nesting level.
+shown to indicate the nesting level. To override this behavior you can use the
+"foldinner" character of the 'fillchars' option.
```

Comment:
```suggestion
shown to indicate the nesting level.  To override this behavior you can use
the "foldinner" character of the 'fillchars' option.
```
See `:h help-writing`:
> Vim help files generally use 2 spaces after a sentence (since they are written
using a fixed-width font and that was the prefered style in the 70s/80s), like
what is described here: https://english.stackexchange.com/a/2602